src/main/webapp/WEB-INF/jsp/index.jsp

```diff
@@ -0,0 +1,111 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <script src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.14/angular.min.js"></script>
+    <title>Welcome Traveller</title>
```

Comment:
What travel app? :D 
No it's from a game and when you open the store it says welcome traveller :P